src/Main.java

```diff
@@ -1,5 +1,13 @@
 public class Main {
     public static void main(String[] args) {
```

Comment:
Je main is wat summier, je had hier wat meer kunnen doen en dat had je denk ik ook wel gekunt. Dat stond echter niet in de opdracht beschrijving, dus dit is wel gewoon voldoende.